### PR TITLE
feat: update colors palette

### PR DESCRIPTION
**Pull request by ej9x**

### CODE REVIEW

**Comment by vorobeez** [comment]
@ej9x Please put `:active` instead of `:focus`.

**Comment by vorobeez** [comment]
@ej9x e2e/__tests__/__image_snapshots__/button-test-js-components-button-common-1-snap.png is cropped. Fix it please

File: src/components/Card/Card.theme.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const [CardBodyTag, cardBodyTheme] = createThemeTag('cardBody', {
 const [CardFooterTag, cardFooterTheme] = createThemeTag('cardFooter', ({ COLORS }: *): * => ({
   root: {
     flexShrink: 0,
-    borderTop: `1px solid ${COLORS.DSM.GREY_COLORS.DSM_LIGHT_GREY_2}`,
+    borderTop: `1px solid ${COLORS.GRAY_20}`,
```
Comment (zouxuoz) [inline src/components/Card/Card.theme.js]: @ej9x What do u think about using border-color alias?

File: src/components/Icon/glyphs/EditPencil.svg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="100%" height="100%" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (zouxuoz) [inline src/components/Icon/glyphs/EditPencil.svg]: @ej9x should we set 100% width/height for icons? or maybe we could remove it and set it via css?

File: src/components/Tooltip/Tooltip.theme.js [hunk 3]
```diff
@@ -4,15 +4,13 @@ import { Z_INDEX } from '../../theme';
 
 const name = 'tooltip';
 
-const [TooltipMessageTag, tooltipMessageTheme] = createThemeTag(`${name}Message`, ({ COLORS, SIZES }: *) => ({
+const [TooltipMessageTag, tooltipMessageTheme] = createThemeTag(`${name}Message`, ({ COLORS, SIZES, FONTS }: *) => ({
   root: {
     // position: 'relative',
     padding: '12px 16px',
     backgroundColor: '#3D4751',
+    ...FONTS.BODY,
```
Comment (lesha1201) [inline src/components/Tooltip/Tooltip.theme.js]: @ej9x There is no field `BODY` in `FONTS`

**Pull request merged**